src/foam/u2/wizard/agents/AlternateFlowAgent.js

```diff
@@ -40,8 +45,25 @@ foam.CLASS({
       }
         
       this.alternateFlow.execute(this.__context__);
-      if ( this.alternateFlow.wizardletId)
-        this.alternateFlow.handleNext(this.__subContext__.wizardController);
+
+      if ( this.alternateFlow.wizardletId ) {
+        if ( this.__context__.wizardController ) {
```

Comment:
Why not import wizardController?

Reply:
fixed, also cleaned up a couple other things